generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -1,1491 +1,1472 @@
 
-          <option value="SM">
-            San Marino
-          </option>
+              <option value="SA">
+                Saudi Arabia
+              </option>
 
-          <option value="ST">
-            Sao Tome and Principe
-          </option>
+              <option value="SN">
+                Senegal
+              </option>
 
-          <option value="SA">
-            Saudi Arabia
-          </option>
+              <option value="RS">
+                Serbia
+              </option>
 
-          <option value="SN">
-            Senegal
-          </option>
+              <option value="SC">
+                Seychelles
+              </option>
 
-          <option value="RS">
-            Serbia
-          </option>
+              <option value="SL">
+                Sierra Leone
+              </option>
 
-          <option value="SC">
-            Seychelles
-          </option>
+              <option value="SG">
+                Singapore
+              </option>
 
-          <option value="SL">
-            Sierra Leone
-          </option>
+              <option value="SK">
+                Slovakia
+              </option>
 
-          <option value="SG">
-            Singapore
-          </option>
+              <option value="SI">
+                Slovenia
+              </option>
 
-          <option value="SK">
-            Slovakia
-          </option>
+              <option value="SB">
+                Solomon Islands
+              </option>
 
-          <option value="SI">
-            Slovenia
-          </option>
+              <option value="SO">
+                Somalia
+              </option>
 
-          <option value="SB">
-            Solomon Islands
-          </option>
+              <option value="ZA">
+                South Africa
+              </option>
 
-          <option value="SO">
-            Somalia
-          </option>
+              <option value="SS">
+                South Sudan
+              </option>
 
-          <option value="ZA">
-            South Africa
-          </option>
+              <option value="ES">
+                Spain
+              </option>
 
-          <option value="SS">
-            South Sudan
-          </option>
+              <option value="LK">
+                Sri Lanka
+              </option>
 
-          <option value="ES">
-            Spain
-          </option>
+              <option value="SH">
+                St. Helena
+              </option>
 
-          <option value="LK">
-            Sri Lanka
-          </option>
+              <option value="PM">
+                St. Pierre and Miquelon
+              </option>
 
-          <option value="SH">
-            St. Helena
-          </option>
+              <option value="SD">
+                Sudan
+              </option>
 
-          <option value="PM">
-            St. Pierre and Miquelon
-          </option>
+              <option value="SR">
+                Suriname
+  
```

Comment:
I's targeting this line https://github.com/mozmar/snippets/pull/93/files#diff-458cd6d86e16cc8924d6fd875daffe51R1337